README.md

```diff
@@ -1,23 +1,39 @@
 # XenoCP
 
+XenoCP is a tool for cleansing mouse reads in xenograft BAMs.
+XenoCP can be easily incorporated into any workflow, as it takes a BAM file
+as input and efficiently cleans up the mouse contamination. The output is a clean
+human BAM file that could be used for downstream genomic analysis. 
+
 ## Getting started
 
-	git clone https://github.com/stjude/XenoCP.git
-	cd XenoCP
-	gradle :xenocp:installDist
+XenoCP can be run in the cloud on DNAnexus at
+https://platform.dnanexus.com/app/stjude_xenocp
 
-Download contaminant genomic reference and update `sample_data/input_data/inputs-local.yml` with the path to the reference data.
+The easiest way to get XenoCP running locally is using Docker, as `docker build`
+creates an image with all of the dependencies:
 
+	git clone https://github.com/stjude/XenoCP.git
+	cd XenoCP
+	docker build -t xenocp .
+	
+	wget -r -np -R "index.html*" -nH --cut-dirs=3 http://ftp.stjude.org/pub/software/xenocp/reference/MGSCv37
+	
+	# Test run on small dataset
 	mkdir results
-	cwltool --outdir results cwl/xenocp.cwl sample_data/input_data/inputs_local.yml
+	docker run \
+	  --mount type=bind,source=$(pwd)/sample_data/input_data,target=/data,readonly \
+	  --mount type=bind,source=$(pwd)/reference,target=/reference,readonly \
+	  --mount type=bind,source=$(pwd)/results,target=/results \
+	  xenocp \
+	  /data/inputs.yml
```

Comment:
Did you confirm that these directions will work?

Reply:
Yes, I walked through the example and corrected the instructions as I went.